casa/OS/Directory.cc

```diff
@@ -301,7 +301,10 @@ void Directory::copy (const Path& target, Bool overwrite,
     String command("cp -r '");
     command += itsFile.path().expandedName() + "' '" +
                targetName.expandedName() + "'";
-    AlwaysAssert (system(command.chars()) == 0, AipsError);
+    int result = system(command.chars());
+    if(result != 0)
```

Comment:
Sorry to nag about this, but casacore uses curly braced in if statements. Please keep the code style consistent.

Reply:
Sure, fair point.